index.js

```
@@ -0,0 +1,75 @@
+const express = require('express');
+const knex = require('knex');
+
+const knexCongif = require('./knexfile');
+
+const db = knex(knexCongif.development)
+
+const server = express();
+
+server.use(express.json());
+
+server.post('/api/cohorts', (req, res) => {
+    const cohort = req.body;
+  
+    db('cohorts')
+      .insert(cohort)
+      .returning('id')
+      .then(ids => {
+        res.status(201).json(ids);
```

You can return a error response here if cohort name already exist.